worlds/yoshisisland/Regions.py

```diff
@@ -0,0 +1,225 @@
+
+        create_region(multiworld, player, locations_per_region, location_cache, "4-1"),
+        create_region(multiworld, player, locations_per_region, location_cache, "4-2"),
+        create_region(multiworld, player, locations_per_region, location_cache, "4-3"),
+        create_region(multiworld, player, locations_per_region, location_cache, "4-4"),
+        create_region(multiworld, player, locations_per_region, location_cache, "Marching Milde's Boss Room"),
+        create_region(multiworld, player, locations_per_region, location_cache, "4-5"),
+        create_region(multiworld, player, locations_per_region, location_cache, "4-6"),
+        create_region(multiworld, player, locations_per_region, location_cache, "4-7"),
+        create_region(multiworld, player, locations_per_region, location_cache, "4-8"),
+        create_region(multiworld, player, locations_per_region, location_cache, "Hookbill The Koopa's Boss Room"),
+
+        create_region(multiworld, player, locations_per_region, location_cache, "5-1"),
+        create_region(multiworld, player, locations_per_region, location_cache, "5-2"),
+        create_region(multiworld, player, locations_per_region, location_cache, "5-3"),
+        create_region(multiworld, player, locations_per_region, location_cache, "5-4"),
+        create_region(multiworld, player, locations_per_region, location_cache, "Sluggy The Unshaven's Boss Room"),
+        create_region(multiworld, player, locations_per_region, location_cache, "5-5"),
+        create_region(multiworld, player, locations_per_region, location_cache, "5-6"),
+        create_region(multiworld, player, locations_per_region, location_cache, "5-7"),
+        create_region(multiworld, player, locations_per_region, location_cache, "5-8"),
+        create_region(multiworld, player, locations_per_region, location_cache, "Raphael The Raven's Boss Room"),
+
+        create_region(multiworld, player, locations_per_region, location_cache, "6-1"),
+        create_region(multiworld, player, locations_per_region, location_cache, "6-2"),
+        create_region(multiworld, player, locations_per_region, location_cache, "6-3"),
+        create_region(multiworld, player, locations_per_region, location_cache, "6-4"),
+        create_region(multiworld, player, locations_per_region, location_cache, "Tap-Tap The Red Nose's Boss Room"),
+        create_region(multiworld, player, locations_per_region, location_cache, "6-5"),
+        create_region(multiworld, player, locations_per_region, location_cache, "6-6"),
+        create_region(multiworld, player, locations_per_region, location_cache, "6-7"),
+        create_region(multiworld, player, locations_per_region, location_cache, "6-8"),
+        create_region(multiworld, player, locations_per_region, location_cache, "Bowser's Room"),
+
+    ]
+    if world.options.extras_enabled.value == 1:
+        regions.insert(68, create_region(multiworld, player, locations_per_region, location_cache, "6-Extra"))

```

Comment:
what is the point of insert here? the order doesn't change anything

Reply:
Locations are sorted by region creation order. It looks nicer to have the Extra stages in the proper placement in the location list than have them all appended at the end. And, I didn't want to create each one individually in the middle of the actual region creation